### PR TITLE
Revert team changes

### PR DESCRIPTION
**Pull request by daehn**
# What's in this PR?

* Update to work with the data model changes made in https://github.com/wireapp/wire-ios-data-model/pull/288.

### This PR depends on:
- [x] A new release of `wire-ios-data-model` after https://github.com/wireapp/wire-ios-data-model/pull/288 has been merged.

### CODE REVIEW

File: Sources/SharingSession.swift [hunk 1]
```diff
@@ -185,17 +185,9 @@ public class SharingSession {
 
     let transportSession: ZMTransportSession
     
-    /// All teams the user belongs to
-    public var allTeams: [Team] {
-        return Array(ZMUser.selfUser(in: userInterfaceContext).teams)
-    }
-    
-    /// Currently selected team, nil for personal team
-    public var currentTeam: Team? = nil
```
Comment (vytis) [inline Sources/SharingSession.swift]: Maybe it's still makes sense to have it as we might want to distinguish if we are sharing to a Team account or not

Comment (daehn) [inline Sources/SharingSession.swift]: Good idea!

Comment (daehn) [inline Sources/SharingSession.swift]: @vytis but not sure we need it in `wire-ios-share-engine`, as we can check the team and display it in the share extension target of the UI project as well (there is only one conversation list now anyway).

Comment (vytis) [inline Sources/SharingSession.swift]: Ah yeah, we have access to `ZMUser.team` anyway. No need to duplicate this.

**Pull request merged**